python/pip_install/pip_repository.bzl

```diff
@@ -119,6 +121,10 @@ and py_test targets must specify either `legacy_create_init=False` or the global
 This option is required to support some packages which cannot handle the conversion to pkg-util style.
             """,
     ),
+    "environment": attr.string_dict(
+        doc = """Environment variables to set before calling the package manager.""",
```

Comment:
nit: maybe mention which ones are typical to set, so users know that this might be what solves their problem?